RandomizableLevers/Resources/languagedata.json

```diff
@@ -232,42 +232,42 @@
   {
     "key": "LEVERNAME.Lever-Crystal_Peak_Tall_Room_Upper",
     "sheet": "UI",
-    "text": "Peaks Tall Room Upper"
+    "text": "Peak's Tall Room Upper"
```

Comment:
It would just be Peak (no possessive), the lever doesn't belong to the area. Compare to other levers with area in the name

Reply:
Well, the counterpoint is that Peak Tall Room Lever is a bad name, which I agree with - but it doesn't make the proposed version any more correct